pkg/osquery/runtime/runner.go

```diff
@@ -290,8 +291,16 @@ func (r *Runner) launchOsqueryInstance() error {
 		"args", strings.Join(o.cmd.Args, " "),
 	)
 
+	if err := os.RemoveAll(paths.extensionSocketPath); err != nil {
```

Comment:
Maybe add a quick comment about why we do this here?

Reply:
good idea